crates/core/src/host/wasm_common/abi.rs

```diff
@@ -54,4 +48,7 @@ pub enum AbiVersionError {
         got: VersionTuple,
         implements: VersionTuple,
     },
+    // TODO: (by 1.0, maybe) remove the parenthetical from this error message
+    #[error("unable to determine ABI of module (may be on an spacetime version < 0.8)")]
```

Comment:
spelling: `a spacetime version`

Reply:
oops, had it on auto-merge. well, it'll get fixed before 1.0 :P